lab/src/Microsoft.ServiceModel.Syndication/tests/SampleTest.cs

```diff
@@ -10,6 +12,76 @@ public static class SampleTest
         [Fact]
         public static void PassingTest()
         {
+           
+        }
+        
+        [Fact]
+        public static void crear()
```

Comment:
>crear [](start = 27, length = 5)

Please give testing function a meaningful name so we can tell what this test is about from the name.